app/locales/ht.json

```diff
@@ -36,53 +42,58 @@
     "nCoV2019_url_info": "Pou plis enfòmasyon sou done pou kat sa a",
+    "news_subtitle": "Li dènye mizajou COVID ki soti nan Otorite Santew e an jeneral ",
     "news_title": "Dènye nouvèl",
     "no_data": "Pa gen okenn done",
     "notification_2_weeks_ago": "2 semenn pase",
-    "notification_data_not_available": "Pa gen done ekspoze ki disponib.",
+    "notification_data_not_available": "Pa gen done ekspozisyon ki disponib.",
     "notification_random_data_button": "Chwazi Otorite Swen Sante",
-    "notification_title": "Ekspozisyon Profile pou 2-Semèn ",
+    "notification_title": "Ekspozisyon Pwofil pou 2 semèn",
     "notification_today": "Jodi a",
     "notification_warning_text": "Si yon Otorite Swen Sante egziste nan zòn ou an ou kapab pran abònman pou jwenn dènye enfòmasyon si gen risk ekspozisyon",
-    "notifications_exposure_format": "{{daysAgo}} jou pas, ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
+    "notifications_exposure_format": "{{daysAgo}} jou pase, ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
     "notifications_exposure_format_today": "Jodi a ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
     "notifications_exposure_format_yesterday": "Yè ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
+    "notifications_no_exposure": "",
     "overlap_found_button_label": "Done Piblik chaje",
     "overlap_no_results_button_label": "Done Piblik chaje",
-    "overlap_para_1": "kote vèt la reprezant lis kote ou ye a\n\nNan ti sèk limyè koulè wouj yo ap reprezante done piblik la",
-    "overlap_title": "Tcheke sipèpoze yo",
-    "push_at_risk_message": "Ou kwaze chemen avèk yon moun ki enfekte COVID-19",
+    "overlap_para_1": "Kote vèt la reprezante lis kote ou ye a\n\nNan ti sèk limyè koulè wouj yo ap reprezante done piblik yo",
+    "overlap_title": "Tcheke sipèpozisyton yo",
+    "push_at_risk_message": "Ou te kwaze chemen avèk yon moun ki enfekte COVID-19",
     "push_at_risk_title": "Ou ka gen risk",
-    "settings_title": "Paramèt",
+    "settings_title": "Tablodbò",
     "share_location_data": "Pataje done lokalizasyon",
-    "show_overlap": "Klike sou yo wè done piblik la",
+    "show_overlap": "Klike sou yo wè done piblik yo",
     "team": "Ekip",
-    "team_para": "Ekip nou an konpoze de yon group ki genyen ladanl epidemyolojis, enjenyè, syantis done, evanjelizasyon vi prive dijital, pwofesè ak chèchè nan anpil gran enstitisyon nan mond lan, tank: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young ak Link Ventures.",
-    "tested_positive_title": "Mwen teste pozitif"
+    "team_para": "Ekip nou an konpoze de yon group ki genyen ladanl epidemyolojis, enjenyè, syantis done, evanjeliS vi prive dijital, pwofesè ak chèchè nan anpil gran enstitisyon nan mond lan, tankou: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young ak Link Ventures.",
+    "terms_of_use": "",
```

Comment:
And one more

Reply:
Will the English version show up if you don't enter anything in these?  I'm guessing not, but wanted to confirm.

Reply:
only if I remove the key. which I can do.